directory.ts

```diff
@@ -59,10 +57,45 @@ export class Directory {
     const user = await this.client.object({objectId: id, objectType: 'user'});
     const { email, picture } = JSON.parse(user.properties.toJsonString());
     return {
-      key: user.id,
+      id: user.id,
       name: user.displayName,
       email,
       picture,
     };
   }
+
+  async insertTodo(todo: Todo) {
+    try {
+      await this.client.setObject({
+        object: {
+          id: todo.ID,
+          type: 'resource',
+          displayName: todo.Title
+        }
+      });
+      await this.client.setRelation({
+        relation: {
+          subjectId: todo.OwnerID,
+          subjectType: 'user',
+          objectId: todo.ID,
+          objectType: 'resource',
+          relation: 'owner'
```

Comment:
Maybe we can change the order to reflect the relation direction as we do in other examples.
e.g.
```
ObjectType
ObjectId
Relation
SubjectType
SubjectId
```